src/app/_utils/sortDigestArticles.ts

```diff
@@ -0,0 +1,11 @@
+import { type DigestArticleData } from '@/types/digestTypes'
+
+export function sortDigestArticles(
+  a: DigestArticleData,
+  b: DigestArticleData,
+): number {
+  if (Number(a.issueNumber) !== Number(b.issueNumber)) {
+    return Number(a.issueNumber) - Number(b.issueNumber)
+  }
+  return a.articleNumber - b.articleNumber
+}
```

Comment:
```suggestion
export function sortDigestArticles(a: DigestArticleData, b: DigestArticleData) {
  if (Number(a.issueNumber) !== Number(b.issueNumber)) {
    return Number(a.issueNumber) - Number(b.issueNumber)
  }

  return a.articleNumber - b.articleNumber
}
```

Also, maybe the function name could be more descriptive? How are we sorting the digest articles? Ascending or descending?

Reply:
how's `sortDigestArticlesAscendingByIssueAndNumber`?